src/pages/PageTwo.vue

```diff
@@ -91,7 +91,7 @@ import EmailPreviewContentful from '@/pages/MonthlyGood/EmailPreviewContentful';
 
 import MoreAboutKiva from '@/pages/MonthlyGood/MoreAboutKiva';
 import KivaAsExpertContentful from '@/pages/MonthlyGood/KivaAsExpertContentful';
-import FrequentlyAskedQuestions from '@/pages/MonthlyGood/FrequentlyAskedQuestions';
+import FrequentlyAskedQuestions from '@/components/MonthlyGood/FrequentlyAskedQuestions';
```

Comment:
I prefer this organization where components in /pages/ are referenced only in the router. Hence why I moved this FAQ compoennt to /components/